install/kubernetes/cilium/values.yaml

```diff
@@ -186,6 +186,8 @@ global:
   endpointRoutes:
     enabled: false
 
+  enableIdentityMark: true
```

Comment:
this should be in the `cilium/charts/config/values.yaml` like the other flags are